DfE.Hexagonal.Common/Repository/IRepository.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DfE.Hexagonal.Common.Repository
+{
+    public interface IRepository<Tid, Tentity>
```

Comment:
Pascal-case (initialising the first letter of every word) would result in the following type variable names:

```csharp
public interface IRepository<TId, TEntity>
```

Do we have a position on varying this for single-letter words, perhaps?

Reply:
Not... really :) will update